### PR TITLE
HWKAPM-626 Test to ensure that jackson is shaded within the agent

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review please? Related to https://issues.jboss.org/browse/HWKAPM-626

File: client/agent/src/test/java/org/hawkular/apm/client/agent/ShadeTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * @author gbrown
+ */
+public class ShadeTest {
+
+    @Test
+    public void testAgentFound() {
+        try {
+            Class.forName("org.hawkular.apm.instrumenter.APMAgent");
```
Comment (pavolloffay) [inline client/agent/src/test/java/org/hawkular/apm/client/agent/ShadeTest.java]: Isn't it better to test returned value? Exception is thrown if there is no such a class.

Comment (objectiser) [inline client/agent/src/test/java/org/hawkular/apm/client/agent/ShadeTest.java]: yeah - only checking that exception not thrown.

**Comment by objectiser** [comment]
@pavolloffay Thanks!

**Pull request merged**